app/javascript/components/widgets/numbers/falling-blocks.jsx

```diff
@@ -185,19 +185,25 @@ class Scene extends React.Component {
     }
     this.setState(state => ({ [block]: state[block] + 1 }));
     // add circles to the world,
-    // Add some .5 'wiggle' to the x position so circles don't fall
-    // directly on top of each other.
+    // randomly pick a point from one wall edge to the other
+    // so that they don't fall directly on top of each other.
     World.add(
       this.engine.world,
-      Bodies.polygon(this.state.width / 2 + (Math.random() - 0.5), -50, 8, 8, {
-        restitution: 0.25,
-        friction: 0.8,
-        render: {
-          sprite: {
-            texture: blockTypes[block],
+      Bodies.polygon(
+        (this.state.width - wallWidth) * Math.random() + wallWidth,
```

Comment:
This is a good example where naming something can improve quality more than a comment:
```
const randomPositionBetweenWalls = (this.state.width - wallWidth) * Math.random() + wallWidth;
....
Bodies.polygon(
  randomPositionBetweenWalls,
```

The `so that they don't fall directly on top of each other` could stay or move into the commits body beneath the first line (better describing the background of the change)